lib/spot-checker.js

```diff
@@ -15,13 +12,15 @@ import {
   UnsupportedHashError,
   validateBlock
 } from '../vendor/deno-deps.js'
+import { MAX_CAR_SIZE } from './constants.js'
```

Comment:
can be removed (see comment above)

Reply:
See response https://github.com/CheckerNetwork/spark-spot-check/pull/2#discussion_r1949408141